Neos.ContentRepository.Core/Classes/Feature/RootNodeCreation/Event/RootNodeAggregateDimensionsWereUpdated.php

```diff
@@ -35,8 +36,9 @@ final class RootNodeAggregateDimensionsWereUpdated implements
     public function __construct(
         public readonly ContentStreamId $contentStreamId,
         public readonly NodeAggregateId $nodeAggregateId,
+        public readonly NodeTypeName $nodeTypeName,
```

Comment:
we cannot update the event payload here - such a change is very breaking. Why do you need this? You can simply copy the node type name from the existing root(s) :)